lib/internal/util/debuglog.js

```diff
@@ -24,10 +25,10 @@ let testEnabled;
 function initializeDebugEnv(debugEnv) {
   debugImpls = ObjectCreate(null);
   if (debugEnv) {
-    debugEnv = debugEnv.replace(/[|\\{}()[\]^$+?.]/g, '\\$&')
-      .replace(/\*/g, '.*')
-      .replace(/,/g, '$|^')
-      .toUpperCase();
+    debugEnv = StringPrototypeReplace(debugEnv, /[|\\{}()[\]^$+?.]/g, '\\$&');
+    debugEnv = StringPrototypeReplace(debugEnv, /\*/g, '.*');
```

Comment:
I think we should wait for V8 9.1 to land on master and use `StringPrototypeReplaceAll` here instead, but non blocking.

Reply:
That's correct. `StringPrototypeReplaceAll` would be the ideal thing here. 
Do we proceed with the changes in this PR or close it for now?